subprojects/test-runner/report-viewer/src/main/kotlin/com/avito/report/internal/model/KtulhuTestCase.kt

```diff
@@ -3,4 +3,4 @@ package com.avito.report.internal.model
 /**
  * Прилетает в prepared_data после анализа совпадения с тестохранилкой если есть test_case_id
  */
-internal data class TestCase(val description: String?)
+internal data class KtulhuTestCase(val description: String?)
```

Comment:
Rename because of the name clashing with test-runner TestCase

Reply:
It is called `Cthulhu`, and we can call it just `AvitoTestCase`